recipes-tag/11-id-chx-analysis/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set year = "2019" %}
 {% set cycle = "1" %}
-{% set version ="2" %}
+{% set version ="3" %}
```

Comment:
I think we should change it to have cycle 2 and version 0 as it's going to be in the 2.0 environments.